app/models/user.rb

```diff
@@ -5,5 +5,5 @@ class User < ApplicationRecord
   has_many :bookings
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
-  validates :name, presence: true
+  # validates :name, presence: true
```

Comment:
this was to test?

Reply:
the validates name crashes the devise. we need to figure out in a new fix.